data/pug.txt

```diff
@@ -317,6 +317,16 @@ outfit "Pug Biodefenses"
 	"unplunderable" 1
 	description "This defense system is designed to release a suite of toxins that are lethal to human beings (and probably to some other species, as well) but which have no effect at all on the Pug."
 
+outfit "Pug Peacekeeping Staff"
+	plural "Pug Peacekeeping Staves"
+	category "Hand to Hand"
+	cost 11000
+	thumbnail "outfit/pug staff"
+	"capture attack" 0.8
```

Comment:
I'm somewhat tempted to say give this a higher capture attack since the defensive capability is pointless when the Pug have biodefenses.

Reply:
How about something >1.6 but <2.8 so that it falls between the Korath rifle and nerve gas?

Reply:
bumped both to 2.3 to give it a unique setup